editoast/src/models/scenario.rs

```diff
@@ -177,8 +174,8 @@ impl List<(i64, Ordering)> for ScenarioWithCountTrains {
     ) -> Result<PaginatedResponse<Self>> {
         let study_id = params.0;
         let ordering = params.1.to_sql();
-        sql_query(format!("SELECT t.*,COUNT(trainschedule.id) as trains_count FROM osrd_infra_scenario t
-            LEFT JOIN osrd_infra_trainschedule trainschedule ON t.timetable_id = trainschedule.timetable_id WHERE t.study_id = $1
+        sql_query(format!("SELECT t.*,COUNT(trainschedule.id) as trains_count FROM scenario t
+            LEFT JOIN train_schedule trainschedule ON t.timetable_id = trainschedule.timetable_id WHERE t.study_id = $1
```

Comment:
```suggestion
        sql_query(format!("SELECT t.*,COUNT(train_schedule.id) as trains_count FROM scenario t
            LEFT JOIN train_schedule ON t.timetable_id = train_schedule.timetable_id WHERE t.study_id = $1
```
Also I find that this `t` isn't a good choice